terraform/dev.backend.tfvars

```diff
@@ -1,2 +1,3 @@
 storage_account_name = "s165d01tfstate"
 key                  = "dev.tfstate"
+resource_group_name  = "s165d01-rg"
```

Comment:
Does this property need adding to the `backend.tfvars` files for other environments?

Reply:
Yes this needs adding to all environment to run terraform commands locally